src/org/zaproxy/zap/extension/bugtracker/RaiseSemiAutoIssueDialog.java

```
@@ -0,0 +1,120 @@
+        super(owner, "bugtracker.dialog.semi.title", dim);
+        this.extension = ext;
+        this.alerts = ext.alerts;
+        bugTrackers = extension.getBugTrackers();
+        initialize();
+    }
+
+    public void setAlert(Set<Alert> alerts) {
+        this.alerts = alerts;
+        initialize();
+    }
+
+    /**
+     * This method initializes this
+     */
+    private void initialize() {
+        this.removeAllFields();
+        this.setTitle(Constant.messages.getString("bugtracker.popup.issue.semi"));
+        addTrackerList(bugTrackers.get(0).getName());
+        updateTrackerFields();
+        
+    }
+
+    public void addTrackerList(String value) {
+        List<String> trackerNames = new ArrayList<String>();
+        for(BugTracker bugTracker: bugTrackers) {
+            trackerNames.add(bugTracker.getName());
+        }
+        this.addComboField(FIELD_TRACKER_LIST, trackerNames, value);
+        for(BugTracker bugTracker: bugTrackers) {
+            bugTracker.setDialog(this);
+            bugTracker.setDetails(alerts);
+        }
+        this.addFieldListener(FIELD_TRACKER_LIST, new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                updateTrackerFields();
+            }
+        });
+    }
+
+    public void updateTrackerFields() {
+        String currentItem = this.getStringValue(FIELD_TRACKER_LIST);
+        for(BugTracker bugTracker: bugTrackers) {
+            if(bugTracker.getName().equals(currentItem)) {
+                this.removeAllFields();
+                addTrackerList(bugTracker.getName());
+                bugTracker.createDialogs();
+                revalidate();
+                repaint();
+            }
+        }
+    }
+
+    @Override
+    public String validateFields() {
+        String currentItem = this.getStringValue(FIELD_TRACKER_LIST);
+        for(BugTracker bugTracker: bugTrackers) {
```

This was my suggestion to allow the user to attempt to fix any problems if the raise of the issue failed, instead of immediately close the dialogue.